civet/src/main.rs

```diff
@@ -1,3 +1,27 @@
+use clap::{arg, Command};
+mod cli;
+
 fn main() {
```

Comment:
Put main in `mod main` not in `mod cli`. We need to separate entrypoint and our code logic.

Reply:
Got it.